client/src/ui/Form.jsx

```diff
@@ -368,121 +374,122 @@ export default class Form extends Component {
-              <p className="form-header-text">Leave a Review</p>
-             {this.props.searchBar && <SearchBar formPopupHandler={this.setCourseIdInSearchBar} isPopup={true} />}
-                  <div className="row form-textbox-row">
-                    <textarea ref={this.textArea} className={"form-input-text" + (err.text || err.textEmpty ? "error" : "")} type="text" value={this.state.text}
-                      onChange={(event) => this.handleTextChange(event)}
-                       />
-                    <div ref={this.emptyMsg} className={err.textEmpty ? "form-field-error" : "hidden"}>Please add text to your review!</div>
-                    <div className={err.text && this.state.text !== "" ? "form-field-error" : "hidden"} id="errorMsg" >Your review contains illegal characters, please remove them.</div>
-                  </div>
-
-
-                  <div className="row">
-                      <div className="col-md-3 col-sm-3 col-xs-3 form-bottom-row-spacing">
-                          <div className="form-label form-professor-label">Professor</div>
-                      </div>
-                      <div className="col-md-8 col-sm-8 col-xs-8 form-select-alignment" ref={this.selectHolder}>
-                          <Select 
-                            className="react-select-container"
-                            classNamePrefix="react-select"
-                            value={this.state.selectedProfessors}
-                            onChange={(professors) => this.handleProfChange(professors)}
-                            isMulti
-                            options={this.getProfOptions()}
-                            ref={this.profSelect}
-                            placeholder="Select"
-                          />
-                      </div>
-                  </div>
+      <div>
+        <div className="form-menu">
+          <form className="form" onSubmit={this.handleSubmit.bind(this)} ref={this.formElement}>
+            <p className="form-header-text">Leave a Review</p>
+            {this.props.searchBar && <SearchBar formPopupHandler={this.setCourseIdInSearchBar} isPopup={true} />}
+            <div className="row form-textbox-row">
+              <textarea ref={this.textArea} className={"form-input-text" + (err.text || err.textEmpty ? "error" : "")} type="text" value={this.state.text}
+                onChange={(event) => this.handleTextChange(event)}
+              />
+              <div ref={this.emptyMsg} className={err.textEmpty ? "form-field-error" : "hidden"}>Please add text to your review!</div>
+              <div className={err.text && this.state.text !== "" ? "form-field-error" : "hidden"} id="errorMsg" >Your review contains illegal characters, please remove them.</div>
+              <div className={!err.text && !err.textEmpty && err.profanity ? "form-field-error" : "hidden"} id="errorMsg" >Your review contains profanity, please edit your response.</div>
```

Comment:
Could be refactored in the future to a map function on a list of errors, whether they're present, and the error message instead of hard coding a div for each one.